packages/website/lib/constants.js

```diff
@@ -1,16 +1,18 @@
-const API = /** @type {string} **/ (process.env.NEXT_PUBLIC_API)
-const MAGIC_TOKEN = /** @type {string} **/ (process.env.NEXT_PUBLIC_MAGIC)
+const API = /** @type {string} **/ (process.env.NEXT_PUBLIC_API);
```

Comment:
no-semis plz. always double check PR diffs for meaningless changes, to spare your reviewers eyeballz.

Reply:
https://github.com/web3-storage/web3.storage/issues/1762